bibharvest/bibfilter_oaipos2inspire.py

```diff
@@ -101,7 +102,11 @@ def main(args):
 
         url = base_url + identifier
         session = requests.session()
-        r = session.get(url)
+        try:
+            r = session.get(url, timeout=60)
+        except ConnectionError, Timeout:
+            register_exception()
+            continue
```

Comment:
Should it rather re-raise exception? Or is the POS bibtasklet idempotent, so the next time it's running it's re-doing everything?